presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -10,6 +10,26 @@ Linux Operating System
 
 * 64-bit required
 * newer release preferred, especially when running on containers
+* adequate ulimits for the user that the Presto process runs as. Currently, 
+  Presto requires a minimum of 4096 open file descriptors, and strongly 
+  recommends at least 8192 when it runs. Many modern linux systems built to 
+  host large applications such as Presto are expected to support much larger 
+  values, on the order of 32k open files descriptors and above, to ensure that
```

Comment:
I believe this is more and we've needed 100k at times. @dain do you remember?

Reply:
Ya... I have no idea why the default in linux isn't like 256k, unless you are on a low end cell phone

Reply:
I've run into issues with 64K once or twice. 100K is generally safe.

Reply:
Maybe we should add a note that the required number of file descriptors scales with the (number of machines)**2 and the workload, unless that is too much detail.

Reply:
Should I just remove the extremely low requirements and leave it at strongly suggesting 128k/256k for the nofile limits, and show that in the code block? Do the nproc lints need to scale with the nofile limits?

Should the limits checked in the code be that low? Do they support a performant system?

<img width="522" alt="Screen Shot 2020-09-03 at 2 39 42 PM" src="https://user-images.githubusercontent.com/1664457/92176501-6f65cd80-edf3-11ea-8edb-553f751dee43.png">